direct/nn/varsplitnet/varsplitnet_engine.py

```diff
@@ -0,0 +1,49 @@
+# coding=utf-8
+# Copyright (c) DIRECT Contributors
+
+from typing import Any, Callable, Dict, Optional, Tuple
+
+import torch
+from torch import nn
+
+from direct.config import BaseConfig
+from direct.nn.mri_models import MRIModelEngine
+
+
+class MRIVarSplitNetEngine(MRIModelEngine):
+    """MRIVarSplitNet Engine."""
+
+    def __init__(
+        self,
+        cfg: BaseConfig,
+        model: nn.Module,
+        device: str,
+        forward_operator: Optional[Callable] = None,
+        backward_operator: Optional[Callable] = None,
+        mixed_precision: bool = False,
+        **models: nn.Module,
+    ):
+        """Inits :class:`MRIVarSplitNetEngine."""
```

Comment:
Yes, what's that? ;-)

Reply:
I don't know why that is there :p I will remove it